src/qibolab/backends.py

```diff
@@ -50,6 +51,33 @@ def __init__(self, platform):
         }
         self.compiler = Compiler.default()
 
+    @property
+    def qubits(self) -> list[Union[str, int]]:
+        """Returns the qubits in the platform."""
+        return list(self.platform.qubits)
+
+    @property
+    def connectivity(self) -> list[tuple[Union[str, int], Union[str, int]]]:
```

Comment:
Same here
```suggestion
    def connectivity(self) -> list[QubitPairId]:
```
(needs `from qibolab.qubits import QubitPairId`).

Reply:
[020f980](https://github.com/qiboteam/qibolab/pull/1076/commits/020f98042c66909c5ad09ee82e7f0f749c35a8a3)